src/app/app-platform/ContentApp.h

```diff
@@ -94,11 +96,30 @@ class ContentAppClientCommandSender
 class DLL_EXPORT ContentApp
 {
 public:
+    struct SupportedCluster
+    {
+        chip::ClusterId clusterIdentifier{ chip::kInvalidClusterId };
+        uint32_t features{ 0 };
+        std::vector<chip::CommandId> optionalCommandIdentifiers;
+        std::vector<chip::AttributeId> optionalAttributesIdentifiers;
+
+        SupportedCluster(chip::ClusterId clusterId, uint32_t feats, const std::vector<chip::CommandId> & commandIds,
```

Comment:
Can you please rename feats into features? and for the struct variable above, you can use mFeatures?

Reply:
I renamed feats -> features, no problem using the same name in the end!